### ISSUE TITLE
[Feature Request][Pro] Multisite Theme builder

### ISSUE THREAD

**Issue by ianlee** [elementor/elementor#8292]
## Prerequisites

- [ x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x] The issue still exists against the latest stable version of Elementor.

## Description

I am about to develop a site for a client with Elementor & Elementor Pro.  The site is to have multiple 'regions' with their own administrators. As such, I am planning on using a multi-site install.  

 We'd like to keep the styles and 'theme' the same using Elementor Pro's theme builder (want to create headers,footers, page, post templates once and apply to all sites), however, I've found that this isn't possible with Elementor Pro as of yet.

Theme builder Templates are not shared between sites, and are not applied across them either.

for similar requests - possibly required prereq before this can be implemented/considered.  
https://github.com/elementor/elementor/issues/1225 
https://github.com/elementor/elementor/issues/2704 
https://github.com/elementor/elementor/issues/2827

**Comment by black-eye** [comment]
+1

**Comment by Abstractionz** [comment]
I would really love this feature.

**Comment by pixallusdev** [comment]
Yes, would be a great addition to manage one header and footer template across all subsites

**Comment by baker-industries** [comment]
+1
